### ISSUE TITLE
Either nothing or ear-splitting static OSX 10.14.6

### ISSUE THREAD

**Issue by doublesupercool** [ExistentialAudio/BlackHole#234]
I am wanting to record from VCV Rack modular music app into Ableton from built-in Audio. After trying to be fancy and set up VCV Rack into Ableton, with no results, I have tried again with the most basic routing:

Aggregate Device with "Built-in Output" and "Black Hole". Built-in Output is the top of the list.
Set the Agg. Dev. to be the AUDIO INPUT (Use this device for sound input)
Set up Logic Pro to use the Agg. Dev. as input.
Play a YouTube video
. . .
Nothing.

I get no indication of signal input into Logic at all.

Even more simple. No aggregate device, just sound input as Black Hole, point recording software to Black Hole . . . no input.
![Screen Shot 2020-12-05 at 09 32 21](https://user-images.githubusercontent.com/26040320/101228564-93c99280-36df-11eb-8eed-846fd4cd42a8.png)
![Screen Shot 2020-12-05 at 09 32 29](https://user-images.githubusercontent.com/26040320/101228574-9deb9100-36df-11eb-9914-ffcb9884fc6e.png)
![Screen Shot 2020-12-05 at 09 32 45](https://user-images.githubusercontent.com/26040320/101228577-a643cc00-36df-11eb-9706-43db4a766886.png)

I usually use an aggregate device of a Behringer Xenyx mixer for output a behringer audio interface for input. I don't know if that is complicating things, but it shouldn't if I am bypassing them.

I get no sound when I monitor with the mixer.  Not really sure what else today. All the documentation and websites say the same thing "Simply make an aggregate device and then point your DAW at the aggregate device". Okay. It no worky. It's definitely a black hole for sound at this point!

**Comment by devinroth** [comment]
Check the volume in AudioMidi setup for BlackHole.

Devin

**Comment by doublesupercool** [comment]
Hmmmm, not sure if this is usual. Black Hole master volume seems up, but built-in output master is greyed out (not selectable) and still no audio out. Thanks for the response, though!

![Screen Shot 2020-12-06 at 08 16 15](https://user-images.githubusercontent.com/26040320/101267420-d0f35a80-379b-11eb-8c15-ffa1ece89d79.png)

![Screen Shot 2020-12-06 at 08 16 22](https://user-images.githubusercontent.com/26040320/101267422-d486e180-379b-11eb-92d8-50f0c55a5a0b.png)

**Comment by devinroth** [comment]
That’s normal.

Run a test for me.


System Output to BH
Logic input to BH

Record in Logic while playing something from Youtube or somewhere else.

Devin

**Comment by doublesupercool** [comment]
System output sent to BH, with BH being set to input/output in audio mid setup
Logic sent to BH for input and output.

Nothing coming in on input 1 in logic, but seems to be recognising output. Not recording anything though.

<img width="658" alt="Screen Shot 2020-12-07 at 13 08 35" src="https://user-images.githubusercontent.com/26040320/101310054-31b08f00-3891-11eb-9075-5370c3e55cdc.png">
<img width="758" alt="Screen Shot 2020-12-07 at 13 10 48" src="https://user-images.githubusercontent.com/26040320/101310057-3412e900-3891-11eb-8cea-3cdb48b427d9.png">

**Comment by devinroth** [comment]
I mean try it without an aggregate or multi output.

Devin

**Comment by doublesupercool** [comment]
Ahh, I posted the wrong thumbnail for that first shot. I was running it straight through Black Hole. I think I have cracked it though. For whatever reason before, the volume wasn't working with Black Hole (using the keys wouldn't change volume etc). This time, I checked the volume in the task bar, which was at 0. Punching it up now has input showing in Logic and the volume up/down keys are working. Recording from YouTube and VCV rack now working! That's good enough for me at this point!

<img width="975" alt="Screen Shot 2020-12-07 at 15 50 06" src="https://user-images.githubusercontent.com/26040320/101333353-db0b7b00-38b9-11eb-86cd-1ea75cfea3b5.png">
<img width="839" alt="Screen Shot 2020-12-07 at 15 51 27" src="https://user-images.githubusercontent.com/26040320/101333366-dcd53e80-38b9-11eb-911d-14b3bd0f17ef.png">
<img width="417" alt="Screen Shot 2020-12-07 at 16 06 45" src="https://user-images.githubusercontent.com/26040320/101333367-dd6dd500-38b9-11eb-8b7f-40a1c012dcc2.png">
